### ISSUE TITLE
[FEATURE] - Remove dummy aliases for Python

### ISSUE THREAD

**Issue by agowa** [Atlas-OS/Atlas#1020]
### What is your feature request regarding to?

Atlas Playbook

### Is your feature request related to a problem? Please describe.

The App Installer has two annoying app execution aliases registered. Namely "python.exe" and "python3.exe", could you just disable them within the Atlas playbook?
They do not execute python and can cause the windows store to open, even though python got installed. Also they're unnecessary and bloated.

### Describe the solution you would like.

Set the App execution alias of "App Installer" to "Off". This setting can be found within "App > Advanced app settings > App execution aliases" of the settings app.

### Describe alternatives you have considered.

does not apply

**Comment by Ast3risk-ops** [comment]
This is a problem on windows in general, Microsoft sucks ass

**Comment by agowa** [comment]
@Ast3risk-ops true, but as Atlas already fixes a lot of other annoyances it kinda would be nice if it was added to the playbook.

**Comment by he3als** [comment]
Might be difficult to do, but a good suggestion.

**Comment by CodingMadness** [comment]
> Might be difficult to do, but a good suggestion.

Just for curiosity, do you guys plan to add it to next release?

**Comment by he3als** [comment]
Not currently, I can't find an official way to programmatically manage app execution aliases.

**Comment by agowa** [comment]
They are managed within: `$env:LOCALAPPDATA\Microsoft\WindowsApps`,  `HKCU:\Software\Microsoft\Windows\CurrentVersion\App Paths`, and `HKLM:\Software\Microsoft\Windows\CurrentVersion\App Paths`

Also see https://learn.microsoft.com/en-us/windows/win32/shell/app-registration#using-the-app-paths-subkey

**Comment by he3als** [comment]
App Execution Aliases don't seem to be managed in those keys. Deleting the file from WindowsApps works but it's a hacky solution that I'd rather not implement, especially as it doesn't update the toggle in Windows Settings.

**Comment by stopspazzing** [comment]
https://stackoverflow.com/questions/24347758/remove-alias-in-script

Using the default installed Powershell version <6 on Windows 11:
```
If(Test-Path Alias:wget) {Remove-Item Alias:wget}
If (Test-Path Alias:wget) {Remove-Item Alias:wget}
wget
```

If Atlas installs and uses newer version 6< then:

```
If(Test-Path Alias:wget) {Remove-Alias -Name wget}
If (Test-Path Alias:wget) {Remove-Alias -Name wget}
wget
```


Untested. Did you want me to test it on a fresh install?

**Comment by agowa** [comment]
@stopspazzing that's a different kind of alias.

**Comment by stopspazzing** [comment]
My bad. Would it be better to use the zipped binaries instead of installing python as it seems deleting registry keys is not on the table this would prevent the app exec alias from being created?

**Comment by RadNotRed** [comment]
> [@stopspazzing](https://github.com/stopspazzing) that's a different kind of alias. We were talking about "App Execution Aliases" not about a [PowerShell alias](https://learn.microsoft.com/en-us/powershell/scripting/learn/shell/using-aliases). They are entirely different things.
> 
> See e.g. the last picture here for what we were talking about: https://www.plothost.com/kb/install-phyton-windows/ This is the alias that we want:
> 
> ![Image](https://github.com/user-attachments/assets/44fe7657-1538-4235-a77a-3823912e5e8a)
> 
> Windows however defaults to some internal garbage that causes the windows store to open when you run `python.exe`. Sadly often enough even after you've installed python...
> 
> Edit: Here you can see the garbage default: https://recursospython.com/guias-y-manuales/no-se-encontro-python-ejecuta-sin-argumentos-para-instalar-desde-microsoft-store/ ![Image](https://camo.githubusercontent.com/041c7f5ad793ea4f8b18e44f7b401fd795cbb596feffac9fdb3168b01067bc8b/68747470733a2f2f7265637572736f73707974686f6e2e636f6d2f77702d636f6e74656e742f75706c6f6164732f323032322f30312f6c697374612d64652d616c6961732d707974686f6e2d77696e646f77732e706e67)

These cannot be turned off or deleted, blame windows for not having winapi for it :) what he did is the right way to remove these aliases. they will NOT turn off, but they will not have the files as long as you dont toggle it off and on

**Comment by agowa** [comment]
@RadNotRed I replied to the "suggestion" by @stopspazzing which is wrong. But looking at it has a redundant `if (Test-Path Alias:python) { Remove-Item Alias:python }; if (Test-Path Alias:python3) { Remove-Item Alias:python3 }`, that only changes runtime state and is unrelated to these aliases here. But it also doesn't hurt, so 🤷

The part that works (like you said already) is the remove the files aka. `Remove-Item -Path \"$env:LOCALAPPDATA\Microsoft\WindowsApps\python*.exe\" -Force -ErrorAction SilentlyContinue`.
